src/shared/views/ProjectView.js

```diff
@@ -510,7 +512,8 @@ class _ProjectHeader extends React.Component<HeaderProps, HeaderState> {
                             fb.analytics().logEvent('starting_tutorial', {
                                 projectType: project.projectType,
                             });
-                            if (project.projectType === LEGACY_TILES) {
+                            if ((project.projectType === LEGACY_TILES)
```

Comment:
this looks very similar to the code above (line 335/336): what's the difference?

Reply:
Indeed. The code above starts the project mapping, while this one goes to the tutorial. I've rewritten this to use a `switch` instead in https://github.com/mapswipe/mapswipe/pull/199/commits/d18ace7876483515004b0df88e4f03435abe6495, so it looks a bit cleaner. We'll probably have to refactor that code after setting up the new Change Detection, as the logic is likely to change a fair bit.